tests/test_fits.py

```diff
@@ -79,5 +80,31 @@ def test_fits_reading():
     im = imageio.imread(compressed)
     assert im.shape == (2042, 3054)
 
+@pytest.mark.skipif("astropy is None")
+def test_fits_get_reader():
+    """Test reading fits with get_reader method"""
+    
+    #Set precedence as normal
+    teardown_module()
+    
+    if IS_PYPY:
+        return  # no support for fits format :(
+
+    import numpy as np
+    from astropy.io import fits
+    sigma = 10
+    xx, yy = np.meshgrid(np.arange(512),np.arange(512))
+    z = (1/(2*np.pi*(sigma**2)))*np.exp(-((xx**2)+(yy**2))/(2*(sigma**2)))
+    img = np.log(z)
+    phdu = fits.PrimaryHDU()
+    ihdu = fits.ImageHDU(img)
+    hdul = fits.HDUList([phdu,ihdu])
+    hdul.writeto('test.fits')
```

Comment:
```suggestion
    hdul.writeto(tmp_path / 'test.fits')
```

We can use above mentioned tmp_path here to create the file inside the temporary directory.

Reply:
```suggestion
    hdul.writeto(tmp_path+'/test.fits')
```

Reply:
Looks like `tmp_path /` is pytest syntax for temporary directories. So, I'll delete my previous suggestion and commit.

Reply:
@astrophysaxist exactly. You can find the docs here: https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture